frontend/src/shared/modules/default-filters/config/organizations.ts

```diff
@@ -0,0 +1,24 @@
+import { IncludeEnum } from '@/modules/organization/config/saved-views/settings/types/IncludeEnum';
+
+export default {
```

Comment:
Missing typescript interface fot his one. 
Also why we dont use existing filter structure here?